factory-containers/build.sh

```diff
@@ -151,6 +158,8 @@ for x in $IMAGES ; do
 			run docker push ${ct_base}:$TAG-$t
 		done
 
+		echo "Build step $((completed+2)) of $total is complete"
+
```

Comment:
Seems like we could simplify and combine this line and line 176 down under the `fi` on line 177?

I know technically we're also doing a docker manifest create in this block, but the customer won't really know that and it's almost like that's being included in step 2.

Reply:
yep. and with the webui checking every 3 seconds, its unlikely this step will even get picked up for showing status.

Reply:
actually - we could remove the 2nd instance of this. its okay that steps are "skipped". and then for 99% of the time we get slightly more granular status. I'll do that in my force-push update.